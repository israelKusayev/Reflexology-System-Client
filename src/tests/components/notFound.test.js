import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../../components/notFound';

test('renders correctly', () => {
  const component = renderer.create(<NotFound />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
