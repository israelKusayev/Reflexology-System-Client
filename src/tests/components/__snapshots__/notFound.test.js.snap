// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <div
    style={
      Object {
        "fontSize": "5em",
        "fontWeight": "bold",
      }
    }
  >
    4
    <span
      aria-label="0"
      role="img"
    >
      ðŸ˜¯
    </span>
    4
  </div>
  <div
    className="bold"
  >
    page not found
  </div>
</div>
`;
